src/web/components/TeamMember/TeamMembersTable.tsx

```diff
@@ -28,15 +30,33 @@ function TeamMembersTableContent({
   onRemoveTeamMember,
   onUpdateTeamMember,
 }: TeamMembersTableProps) {
+  const { LoggedInUser } = useContext(CurrentUserContext);
+  const { participant } = useContext(ParticipantContext);
+  const userRolesForCurrentParticipant = LoggedInUser?.user?.participants?.find(
```

Comment:
nit: maybe this could also be extracted into a helper method

Reply:
Extracted it into the same helper method as above - `isUserAdminorSupport`